pupil-spa/src/app/services/device/device.service.spec.ts

```diff
@@ -37,9 +41,20 @@ describe('DeviceService', () => {
   });
 
   it('captures device info to localStorage',  async () => {
+    // test setup
+    service.deleteDeviceCookie();
+    service.setupDeviceCookie();
+
+    // exec
     await service.capture();
+
+    // tests
     const deviceInfo = storageService.getDeviceData();
     expect(deviceInfo).toBeTruthy();
```

Comment:
when you pull in the tslib lint update PR, these type of assertions will fail linting.

please use `toBeInstanceOf`, `toBeDefined` and `toBe(true)` instead

Reply:
Or possibly not yet?  Master was merged in yesterday, and that included the eslint changes.   This is the SPA, and I see a couple of  issues:

1. Neither `yarn test` or `yarn test-single` do a lint run
2. `yarn lint` actually calls `ng lint` which passes

The SPA does not seem to have been updated to run `eslint` yet.

Reply:
I've changed it to `toBeDefined()` 😄